resources/functions/user-management/index.py

```diff
@@ -14,28 +15,40 @@
 logger = Logger()
 app = APIGatewayRestResolver()
 
+event_bus = boto3.client('events')
+eventbus_name = os.environ['EVENTBUS_NAME']
+event_source = os.environ['EVENT_SOURCE']
+user_created_detail_type = os.environ['USER_CREATED_DETAIL_TYPE']
+user_deleted_detail_type = os.environ['USER_DELETED_DETAIL_TYPE']
+user_enabled_detail_type = os.environ['USER_ENABLED_DETAIL_TYPE']
+user_disabled_detail_type = os.environ['USER_DISABLED_DETAIL_TYPE']
 idp_name = os.environ['IDP_NAME']
```

Comment:
why is the user management for control plane is publishing these event? Is there anything getting triggered based on these events.

Reply:
this was a mistake. I've removed this update.